src/components/Preferences/EmptyLocalIdentities.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react'
+import PropTypes from 'prop-types'
+import styled from 'styled-components'
+import { EthIdenticon, IdentityBadge, breakpoint } from '@aragon/ui'
+import { getEmptyAddress } from '../../web3-utils'
+import Import from './Import'
+
+const EmptyLocalIdentities = ({ onImport }) => (
+  <Wrap>
+    <Title>Start adding labels</Title>
+    <Paragraph>
+      You can add labels by clicking on the{' '}
+      <WrapBadge>
```

Comment:
This badge renders a bit broken on Firefox:

<img width="400" alt="image" src="https://user-images.githubusercontent.com/36158/54933445-ecadc300-4f1c-11e9-8843-cdd47a8504c5.png">